MediaBrowser.Model/Configuration/LibraryOptions.cs

```diff
@@ -93,6 +93,12 @@ public LibraryOptions()
                 }
             }
 
+            if (type == "Trailer")
+            {
```

Comment:
Not sure how I feel about this, but you should use nameof instead of hardcoding the type name

Reply:
I agree this is more hackish than the other workarounds I added. I was going to try adding a field to `TypeOptions` to add a secondary type so the `Video` option could also match `Trailer` but that looked like it was going to require a change to `LibraryTypeOptionsDto`, which seemed like larger scope than I wanted to hit with this.

If the other changes in this pull request look good I'd be happy to yank this part out and say so in a comment in the related issue.